yarn.lock

```diff
@@ -2981,6 +3018,32 @@
     ts-dedent "^1.1.1"
     util-deprecate "^1.0.2"
 
+"@storybook/api@6.0.26":
```

Comment:
Would you mind manually deduplicating some of these installs?

Reply:
Could I just remove the dependencies from packages/kbn-storybook/package.json?

Reply:
I updated to the latest versions of everything which should take care of this.

Reply:
Sorry, what I was thinking was deleting the entires from the yarn.lock file which are triggering duplicates manually, like deleting all `@storybook/api` entries in this example, and then rerun bootstrap which will cause all versions of that package to use the same version

Reply:
It's possible the changes you've made had the same effect, and we're getting pretty close to using a single package.json file, so if this has the same effect it works for me